inc/luos.h

```diff
@@ -63,6 +63,10 @@ uint16_t Luos_NbrAvailableMsg(void);
 uint32_t Luos_GetSystick(void);
 error_return_t Luos_TxComplete(void);
 void Luos_Flush(void);
+uint16_t Luos_IDFromType_localhost(uint16_t type);
+bool Luos_IsNodeDetected(void);
+void Luos_SetFilterState(uint8_t state, uint16_t id);
```

Comment:
As I understand you need to specify an ID because you need to redirect all the messages received to this specific service.
But The ID is a network thing, not an internal value.
Users will struggle to get it and then you will struggle to use it on your code because you will have to find back the good local service having this ID.
Also an ID can change and lead to BIG trouble here.
I think you should consider asking a `service_t*` instead. This is how we make it in all the other functions with this kind of concern.

Reply:
I agree ✅